.github/workflows/mvn-deploy.yml

```diff
@@ -0,0 +1,29 @@
+name: Publish package to the Maven Central Repository
+
+on:
+  release:
+    types: [created]
+
+jobs:
+  Publish:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up Maven Central Repository
+        uses: actions/setup-java@v2
+        with:
+          java-version: '11'
+          distribution: 'temurin'
+          server-id: ossrh
+          server-username: ${{ secrets.OSSRH_USERNAME }}
+          server-password: ${{ secrets.OSSRH_TOKEN }}
+          gpg-private-key: ${{ secrets.MAVEN_GPG_PRIVATE_KEY }}
+          gpg-passphrase: ${{ secrets.MAVEN_GPG_PASSPHRASE }}
+      - name: Publish package
+        run: |
+          mvn --settings .travis/settings.xml org.codehaus.mojo:versions-maven-plugin:2.1:set 1>/dev/null 2>/dev/null
+          mvn clean deploy --settings .travis/settings.xml -Dgpg.passphrase=${{ MAVEN_GPG_PASSPHRASE }} -DskipTests=true -B -U -P release
```

Comment:
did you test this? I think there needs to be some seperation between commands?

Reply:
Line 24 and 25 are executed sequentially, if you mean that. Besides that, I took it from the previous script we successfully used for travis:

```shell
if [ ! -z "$TRAVIS_TAG" ]
then
    echo "on a tag -> set pom.xml <version> to $TRAVIS_TAG"
    mvn --settings .travis/settings.xml org.codehaus.mojo:versions-maven-plugin:2.1:set -DnewVersion=$TRAVIS_TAG 1>/dev/null 2>/dev/null
else
    echo "not on a tag -> keep snapshot version in pom.xml"
fi

mvn clean deploy --settings .travis/settings.xml -Dgpg.passphrase=$GPG_PASSPHRASE -DskipTests=true -B -U -P release
```